tidy3d/plugins/design/result.py

```diff
@@ -330,6 +351,21 @@ def delete(self, fn_args: Dict[str, float]) -> Result:
         new_coords.pop(index)
         new_values.pop(index)
 
+        # ParticleSwarm optimizer doesn't work with updated_copy
```

Comment:
curious why not? what's the error?

Reply:
If I remember rightly it's because parts of the PSO object are pickled which updated_copy fails to handle.